crypto/bls12381_utils.c

```diff
@@ -84,7 +85,10 @@ prec_st* init_precomputed_data_BLS12_381() {
 // Initializes Relic context with BLS12-381 parameters
 ctx_t* relic_init_BLS12_381() { 
     // check Relic was compiled with the right conf 
-    if (ALLOC != AUTO) return NULL;
+    assert(ALLOC == AUTO);
+
+    // sanity check of Relic constants the package is relying on
+    assert(RLC_OK == RLC_EQ);
```

Comment:
👍 do we need to do this for other places that we use Relic?

Reply:
That function is called under the package main `init()` function, so it's guaranteed to be called whenever the crypto package is used.